html-7/js/HTML-7.js

```diff
@@ -0,0 +1,10 @@
+let links = document.getElementsByTagName('a');

```

Comment:
This can be done in css with :hover selector which is the better way to do that.

pseudocode:
```
a:hover {
    background: white;
    color: green;
}
```

Reply:
task was to do that in javascript :)